### PR TITLE
Updated to reflect VSTS reband to Azure DevOps

### PR DESCRIPTION
**Pull request by rayhogan**
Visual Studio Team Services has now been rebranded to Azure DevOps. I have updated the Readme file to reflect this change. This will ensure that users will be able to follow the documentation correctly.

### CODE REVIEW

**Comment by yacaovsnc** [comment]
Thanks for your pull request!  Can you please also update this [file](https://github.com/Microsoft/vsts-ibm-websphere-extension/blob/master/docs/vsts-README.md)?

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 <td width="140px" style="text-align: center;"><img src="websphere_icon.png" style="max-width:100%" /></td>
-<td><strong>Visual Studio Team Services Extension for IBM WebSphere Application Server</strong><br />
+<td><strong>Azure DevOps Extension for IBM WebSphere Application Server</strong><br />
 <i>Provides deployment task that automatically installs/updates enterprise applications in WebSphere Application Servers.</i><br />
 <a href="https://marketplace.visualstudio.com/items/ms-vsts.ibm-websphere">Install now!</a>
 </td>
 </tr></table>
 
-# Visual Studio Team Services Extension for IBM WebSphere Application Server
+# Azure DevOps for IBM WebSphere Application Server
```
Comment (yacaovsnc) [inline README.md]: Seems the word 'Extension' is removed by mistake.

**Comment by yacaovsnc** [comment]
Thank you for your contribution!

File: README.md (hunk 1 above)
Comment (rayhogan) [reply README.md]: Apologies for the mistake, thanks for bringing it to my attention. I have updated the file now.

**Pull request merged**